torch/csrc/profiler/collection.cpp

```diff
@@ -296,6 +296,19 @@ void ThreadLocalSubqueue::TorchOpStorage::materialize(
     }
   }
 
+  // `AccumulateGrad` is an important marker for profile analysis; however the
+  // annotation relies on `c10::demangle` which is platform dependent. In
+  // particular, Windows will add a "struct " prefix.
+  const std::string accumulate_grad = "torch::autograd::AccumulateGrad";
+  const std::string windows_pattern = std::string("struct ") + accumulate_grad;
```

Comment:
Is this pattern only important for AccumulateGrad, or should it be generalized elsewhere?

Reply:
It might crop up again, but in general it should be pretty limited. We generally try not to rely on demangled names; AccumulateGrad just happens to be very special.